test/helpers/helpers.ts

```diff
@@ -0,0 +1,43 @@
+import { LoginInput, UserInput } from "../../src/graphql/schema";
+import bcrypt from "bcrypt";
+
+export const hashPassword = async (password: string) => {
+  const saltRounds = 10;
+  const hashedPassword = await bcrypt.hash(password, saltRounds);
+  return hashedPassword;
+};
```

Comment:
Essa função está sendo usada na mutation de criação de usuário? Só pra checar porque códigos da pasta test não deveriam ser chamados dentro da src.

Mas se não for o caso, talvez seja mais interessante reaproveitar uma função da src que já faz isso do que criar outra aqui nos testes, né?

Reply:
sim, acabei fazendo isso. Essa função estava na src e copiei ela para a test. Mas acabei mudando de ideia e exportei a função da src para ser usada nos testes.